src/util.js

```diff
@@ -16,6 +18,46 @@ export function deepMerge(dest, src) {
   }
 }
 
+/**
+ * Helper function for chaining dependent observables.
+ *
+ * @param {Function} factory - Function that creates an observable.
```

Comment:
let's use `createDependentObservable` instead of `factory`

and use `{function(lastMessage): Observable}` as a type

double space instead of dash between param name and description

"Function that is passed the last message emitted by the source observable and returns a new observable"